com.github.mednaffe.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.mednaffe</id>
```

Comment:
The app-id should be com.github.amatcoder.mednaffe

Reply:
Hmm, but id from application is com.github.mednaffe:
https://github.com/AmatCoder/mednaffe/blob/master/src/mednaffe.c#L48

Do I need to patch it?

Reply:
Yes, I need patch it. 
If not it give me errors with DBus.

Reply:
yes. The namespace on github is per user/organisation.